API-strategie-modules/access-control/access-control.md

```diff
@@ -155,15 +156,25 @@ Using tokens, a distinction is made between authorized and non-authorized servic
 
 In case the proper headers are not sent, then there are no authentication details available and a status error code `403 Forbidden` is returned.
 
-<div class="rule" id="api-52">
-  <p class="rulelab"><strong>API-52</strong>: Use OAuth 2.0 for authorization with rights delegation</p>
+**How to test**
+Testing is conditional on a correct Open API Specification(OAS) being present as well as an access token being provided by the API provider to the test client. If this is the case an automated test client can test if the access control method works according to this rule and the method specified in the OAS spec is indeed implemented. Negative tests are out of scope as there are to many variables to account for when testing other (not allowed) methods for providing tokens.
```

Comment:
Wellicht goed om te verduidelijken dat we het `security` object uit de OAS gebruikt wordt. Daarbij welke `type`s toegestaan zijn: https://spec.openapis.org/oas/v3.0.0.html#fixed-fields-22